tests/cli/test_registry.py

```diff
@@ -172,3 +176,29 @@ def test_list_project_dimension_query_names(cached_registry):
     assert "subset: commercial_subsectors2 residential_subsectors" in result.stdout
     assert "supplemental: all_subsectors commercial_subsectors" in result.stdout
     assert "supplemental: all_geographies census_division census_region state" in result.stdout
+
+
+def test_register_standard_scenarios(tmp_registry_db):
+    """Test registration of the StandardScenarios project."""
+    _, tmpdir, db_name = tmp_registry_db
+    runner = CliRunner(mix_stderr=False)
+    result = runner.invoke(admin_cli, ["create-registry", db_name, "-p", str(tmpdir), "--force"])
+    assert result.exit_code == 0
+
+    project_config = STANDARD_SCENARIOS_PROJECT_REPO / "dsgrid_project" / "project.json5"
+
+    result = runner.invoke(
+        cli,
+        [
+            "--database-name",
+            db_name,
+            "--offline",
+            "registry",
+            "projects",
+            "register",
+            str(project_config),
+            "--log-message",
+            "log",
+        ],
+    )
+    assert result.exit_code == 0
```

Comment:
I agree that this test isn't problematic per se. How awkward would it be later to replace the standard scenarios submodule with something else?

Reply:
The dilemma is this: What is the best way to get full test coverage of our code? Do we create fake projects that have everything that our real projects have? Or do we just clone those projects in the tests? Making a git submodule is the easiest way to keep the “test” data in sync with the code. It could be a simple clone as long as we track the branch name. We should probably try to register every project that is open source. A secondary benefit is that it provides extra checking on the project repository. We have introduced breakages many times.